### ISSUE TITLE
no-member false-positive for monkey patching in transitive import

### ISSUE THREAD

**Issue by vlovich** [PyCQA/pylint#2794]
### Steps to reproduce
1. cat monkey.py
```
import monkey2
```
2. cat monkey2.py
```
import logging
logging.VERBOSE = 5
```
3. cat test.py
```
import monkey
```

### Current behavior
************* Module test
test.py:4:6: E1101: Module 'logging' has no 'VERBOSE' member (no-member)

### Expected behavior
No error like if test.py imports monkey2 directly.

pylint 2.3.1
astroid 2.2.3
Python 3.7.2 (default, Feb 13 2019, 15:31:10) 
[Clang 10.0.0 (clang-1000.11.45.5)]

**Comment by PCManticore** [comment]
I'm sorry but I don't think `pylint` needs to support monkeypatching like this case, it complicates the inference engine for little benefit. I would suggest either disabling the error locally or using `--generated-members` to make sure this value is understood.
